### PR TITLE
[ADD] Added calendar_resources module.

### PR DESCRIPTION
**Pull request by bjoliveau**
This module has been developed by Jordi @foutoucour 

I've just discovered it hasn't been merged on OCA and think it's a nice feature, used by the customer who financed it.

This module add new features to facilitate your resources management with your meetings :
- improvement in resource.resource native object (image, note and boolean choice for using it in your calendars)
- add a link between your calendars and your resources,
- switch easily your calendar's views with attendees and resources,
- add a new menu entry to display event by resources in several views (calendar, list and form)

Note that the customer don't expect for this step :
- any constraint between resources reservations in calendar,
- any interface with calendar_google for the 'resource' object.

### CODE REVIEW

**Comment by bjoliveau** [comment]
Hey @oca-clabot 

Jordi Riera github login @foutoucour

**Comment by max3903** [comment]
CLA is ok. The commit were done with his Savoir-faire Linux address which is no longer linked to his Github account.

File: calendar_resources/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+{
+    'name': 'Calendar Resources',
+    'version': '0.1',
+    'author': 'Savoir-faire Linux',
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'license': 'AGPL-3',
+    'category': 'CRM',
+    'summary': 'New features to facilitate resources management with meetings',
+    'description': """
```
Comment (hbrunn) [inline calendar_resources/__openerp__.py:32]: move this to README.rst with OCA template https://github.com/OCA/maintainer-tools/tree/master/template/module

File: calendar_resources/models/calendar.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import logging
+_logger = logging.getLogger(__name__)
```
Comment (hbrunn) [inline calendar_resources/models/calendar.py:22]: unused

File: calendar_resources/models/resource.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import logging
+_logger = logging.getLogger(__name__)
```
Comment (hbrunn) [inline calendar_resources/models/resource.py:22]: unused

File: calendar_resources/views/resource_view.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="resource_resource_image_form" model="ir.ui.view">
+            <field name="name">Resources: form with image and more</field>
+            <field name="model">resource.resource</field>
+            <field name="inherit_id" ref="resource.resource_resource_form"/>
+            <field name="arch" type="xml">
+                <field name="name" position="replace"/>
+                <form position="after">
```
Comment (hbrunn) [inline calendar_resources/views/resource_view.xml:10]: why `after`?

**Comment by hbrunn** [comment]
```
./calendar_resources/models/resource.py:37:1: W391 blank line at end of file
./calendar_resources/__openerp__.py:35:80: E501 line too long (89 > 79 characters)
./calendar_resources/__openerp__.py:36:80: E501 line too long (112 > 79 characters)
./calendar_resources/__openerp__.py:39:80: E501 line too long (95 > 79 characters)
```

**Comment by dreispt** [comment]
Looking at the dependencies, this module does not require CRM to work.
Could it serve other use cases in instances without CRM installed? Is so, maybe `base_resource_calendar` would be a better name.

**Comment by pedrobaeza** [comment]
@bjoliveau, can you please attend review comments?

**Comment by bealdav** [comment]
@bjoliveau  Please, could you rename your module in singular form as defined by guidelines.

Thanks

**Comment by rafaelbn** [comment]
Hi @bjoliveau is it not needed we can close this PR. If needed you should review comments and we will go ahead. Thanks

**Pull request closed without merging**